src/main/resources/i18n/strings.properties

```diff
@@ -63,6 +63,13 @@ addvaultwizard.new.validCharacters.message=The vault name may contain the follow
 addvaultwizard.new.validCharacters.chars=Word characters (e.g. a, ж or 수)
 addvaultwizard.new.validCharacters.numbers=Numbers
 addvaultwizard.new.validCharacters.dashes=Hyphen (%s) or underscore (%s)
+addvaultwizard.new.enableExpertSettings.checkbox=Enable expert settings
+addvaultwizard.new.shorteningThreshold.title=Limit the length of encrypted file names.
+addvaultwizard.new.shorteningThreshold.tooltip=Open the documentation to learn more.
+addvaultwizard.new.shorteningThreshold.namePrompt=36-220
```

Comment:
remove translation key. These numbers must not be part of the translation.

Reply:
done